README.md

```diff
@@ -1,2 +1,49 @@
 # lev-api-stress
-Stress testing for the `lev-api`.
+Stress testing for the [LEV API], based on the [Artillery.io] tool.
+
+
+## Running the test
+The performance tests can be started with the following command:
+```shell script
+./entrypoint.sh <path-to-config-file>
```

Comment:
Is there no config file included that could be referenced?

Reply:
There is now an example/template file. Fixed